sale_stock_reconcile_valuation_kit/tests/test_sale_stock_reconcile_valuation_kit.py

```diff
@@ -88,7 +88,36 @@ def setUp(self):
 
     def test_account_kit_reconciled(self):
         lines = self.invoice.line_ids.filtered(
-            lambda x: x.account_id.id == self.stock_output_account.id
+            lambda x: x.account_id == self.stock_output_account
         )
         for line in lines:
-            self.assertTrue(line.reconciled)
+            self.assertTrue(line.reconciled, "Invoice line is not reconciled")
```

Comment:
I don't understand the tests you added below. At this line, line.reconciled has been `True`.
And they will not be counted because of the filter:
[product_account_moves = move.line_ids.filtered(
                        lambda line,
                        prod=prod,
                        account=product_interim_account: line.product_id == prod
                        and line.account_id == account
                        and not line.reconciled
                    )](https://github.com/Kimkhoi3010/sale-workflow/blob/cec27c7ddfd4b861caa5078975344c4b2444eac1/sale_stock_reconcile_valuation_kit/models/account_move.py#L43C1-L43C48)